crypto-primitives/src/crh/mod.rs

```diff
@@ -29,7 +29,7 @@ pub trait CRHScheme {
         + Default
         + CanonicalSerialize
         + CanonicalDeserialize;
-    type Parameters: Clone + CanonicalSerialize + CanonicalDeserialize;
+    type Parameters: Clone + CanonicalSerialize + CanonicalDeserialize + Sync;
```

Comment:
this seems to be required after patching to include https://github.com/arkworks-rs/std/pull/48.
Now while adding `Sync` to these types here is innocuous, I'd like to understand how the change in `std` causes an explicit requirement on `Sync` here